src/applications/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -28,8 +29,15 @@ class FacilityDetail extends Component {
   componentDidUpdate(prevProps) {
     const justLoaded =
       prevProps.currentQuery.inProgress && !this.props.currentQuery.inProgress;
-
     if (justLoaded) {
+      if (
+        hasVADomain(
+          this.props.facility.attributes &&
+            this.props.facility.attributes.website,
+        )
+      ) {
+        window.location.replace(this.props.facility.attributes.website);
```

Comment:
We may want to use ```window.location.assign()``` here.  Replace will not allow to use the back button to navigate to the previous page where assign will.  Sometimes using assign can result in a security exception.  I am not sure if it will in this case, but its worth a look.